tests/unit_tests/blas/include/reference_blas_templates.hpp

```diff
@@ -1103,13 +1166,13 @@ void rot(const int *n, double *x, const int *incx, double *y, const int *incy, c
 template <>
 void rot(const int *n, std::complex<float> *x, const int *incx, std::complex<float> *y,
          const int *incy, const float *c, const float *s) {
-    csrot_(n, (void *)x, incx, (void *)y, incy, c, s);
+    onemkl_csrot(n, (void *)x, incx, (void *)y, incy, c, s);
```

Comment:
I have reservations about this function name. It looks like a onemkl function. Maybe we can call it *_wrapper?

Reply:
renamed to csrot_wrapper